### PR TITLE
Add some PHPDoc type annotations

### PR DESCRIPTION
**Pull request by oliverklee**
This will help static type analysis tools.

### CODE REVIEW

**Comment by sabberworm** [comment]
Really cool, thanks. Is there some check we could install into the CI build that ensures everything remains annotated?

**Comment by oliverklee** [comment]
> Is there some check we could install into the CI build that ensures everything remains annotated?

Yes, I will add PHPStan checks later to make sure the types are correct.

We can also add PHP_CodeSniffer checks for checking for the presence of type annotations, but I'd like to avoid that as we probably won't need many of them once we've updated the PHP requirement and we can use more of the native type declarations.

**Comment by westonruter** [comment]
> We can also add PHP_CodeSniffer checks for checking for the presence of type annotations, but I'd like to avoid that as we probably won't need many of them once we've updated the PHP requirement and we can use more of the native type declarations.

Updating the PHP requirement detrimental for WordPress which still supports PHP 5.6. If PHPDoc can be used until WordPress [increases the minimum](https://make.wordpress.org/core/2020/08/24/proposal-dropping-support-for-old-php-versions-via-a-fixed-schedule/), this would be ideal. Otherwise, the [dozens of WordPress plugins](https://wpdirectory.net/search/01F3NM1ZE9PHFQJ957XEXBJJ7C) that use PHP-CSS-Parser would be forced to fork the package.

**Comment by oliverklee** [comment]
@westonruter @sabberworm I'd really appreciate this PR getting reviewed and merged as it blocks other things I'd like to work on that also touch the code. Thanks! :heart:

File: lib/Sabberworm/CSS/Property/Charset.php [hunk 1]
```diff
@@ -10,11 +10,21 @@
...
+	/**
+	 * @var string
+	 */
 	private $sCharset;
+
+	/**
+	 * @var int
+	 */
 	protected $iLineNo;
 	protected $aComment;
```
Comment (westonruter) [inline lib/Sabberworm/CSS/Property/Charset.php:22]:
```suggestion

	/**
	 * @var array
	 */
	protected $aComment;
```

**Review by westonruter — APPROVED** [review]
Looks good! Just one additional phpdoc tag that I saw could be added as well.

**Review by sabberworm — APPROVED** [review]

**Pull request merged**